dot/core/runtime_test.go

```diff
@@ -66,3 +71,72 @@ func TestRetrieveAuthorityData(t *testing.T) {
 		t.Fatalf("Fail: got %v expected %v", auths, expected)
 	}
 }
+
+func TestValidateBlock(t *testing.T) {
```

Comment:
none of the tests within runtime changed, just runtime related tsts moved to runtime